### ISSUE TITLE
started Wildfly server shows as not started

### ISSUE THREAD

**Issue by linkStrack** [apache/netbeans#5357]
### Apache NetBeans version

Apache NetBeans 16

### What happened

After install some plugings netbeans Dont show the correct status of the serv,er, wildfly. Always show the server like dont started.

### How to reproduce

Intalll plugins of the market, like: JAVA FX, PHP, 

### Did this work correctly in an earlier version?

Apache NetBeans 16

### Operating System

Windows 11

### JDK

17

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

Yes

### Code of Conduct

Yes

**Comment by mbien** [comment]
what is the log line of your started wildfly server? For example:
```
00:31:21,711 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: WildFly Full 27.0.1.Final (WildFly Core 19.0.1.Final) started in 4391ms - Started 460 of 691 services (376 services are lazy, passive or on-demand) - Server configuration file in use: standalone-full.xml
```
also where do you see it as not-started? The buttons in the output window or something else?

NB 16 had some fixes in this area #4694

**Comment by asbachb** [comment]
Closing due to no further response. Might be related to #5947 

Feel free to reopen with more details if this differs from the issue mentioned above.
